credentials/apps/edx_credentials_extensions/edly_credentials_app/api/v1/views/tests/test_views.py

```diff
@@ -0,0 +1,89 @@
k033ZTKysqKSrt5/PHH06FDh2y2IKnTp5DSpGzkyJEF3Q85uWImLlkSe/fudf5tnv78889IB33ooYfStddeW9TalHHVVVdFZ9nDhw97jSWpKweE7O5SB0LL6JYWhSuuuCJ98cUX3TZ1lB1Xdq7fe++9SJc599xzC04LpUaQ1NDHHnssTmetE5TUlTbMmD23dOnSyHwoJGuC0ywyLki/51Tq448/jpR8N83+u2azJnFNmBdYW1sbA+D79+8fGT2FXPPmqbzUeLJJTB0776UntpLUDQJCEOxwWpVtkWCBOH78eLd6k//666/0xhtvpBkzZsTDTbagOddhzTNnzkyfffaZDzeSurSvvvoqGo/06tWrqPsmJ4ejRo1Ks2fPjiZonIR1x81JZtB+8sknaeXKlXFduSb5NvTJpl+/fvF1GQ9isx9J6oYBIbu6tI7OVmNAMESNCHOGuuJizM/EqIjff/89ffjhh6m6ujoNHTo02moXWnuRKcK/+OKL08KFCyM11IVWUndx8uTJaGjCCRbjdwrpvNx8Y5LgkuHn99xzT5yMcULGPZt01a6wycY6RLMWNl/5udg8XLJkSTR3Ky8vT+edd15R3UHPzFbhPbnmmmviWhLA8375eytJ3TQgBMEKC86/LcjUh7z//vtdprCcIDCz6zp16tTofFfMQ0vzQnzqOBhBQfMZ/6eR1F0RrDFmgnETl156aavcY5t3Kx0/fnw0UKHjKQHo0aNHO0WDrr///js1NDTEGvTWW2/F3N9HHnkkAugBAwa02nU6c5NyxIgRkY67Z88eyxYkyYDwfxfsV199NfXs2TOn8QizZs2KQKcz1RfwM3JCRyoop4A8nDDzj5bphdYEnrnjSg0HzWZY3H/66Sf/Z5GkZvXYdKp88cUXI3OimBEILQU73M+pP5w4cWI07Vq0aFF68803IyDl3k/XbE7CeC2sCWxuso4RQBaT/cLn8jX4ejTE4euz3rDpyAnm999/H81y3nnnncgYmT59egRmV155ZbzmYrNRsnWyZn1j3Wb0xPbt2yPTx26hkmRAeFYsYiygue5KXnDBBdFtk0WuFGcSseCRdsODALWA/GyccLb2Qwhfj/lOTz31VNq/f78LrSTlkJ3Bffnmm2+O1MXWDoRaQuDF/XrChAnp0UcfjZPF5557Lk7n2BRdv359dEvdsGFDevfdd2PmHiMcwJgh/szfb9y4MV4/H8/IDNIuX3rppTh5q6mpSZWVldGlk0CMRjnt9fM1D46pfSfzhddHUOrvnSQZEOaEgv3bb78956CQ2sKLLroo3XXXXem1117r0JmF7NCy88nCTedUOoLecMMNafDgwbE4tsWCywkjiy2jKOzGJkn5N0vZvXt3evLJJ2NmXluckhWS7cFGX48ePWK2Ip03QeBKIxf+nvt/sY3G2uJ1cw0ZPcGp6MGDB61blyQDwsKDQnYV861foJ6DhZI0mDlz5sQOKydmpE6SqsMpYqF1HQR7BJuc+DGugVNJuqGtXr06dmRvueWW6JLGa+Z1tNVDBUX9FRUVsavM8F8HyUtS62zosam2b9++uL+OHj06OjO3RR1dV0GJB0PjaTpDtg4dw7mG3XVElCQZELbBri31BtTEFbNbSbrMddddFwEbJ3YUzFM/sWLFiki14VQR1FWAdBx2NletWhV1Jk8//XTM7CMFh5NLHhKGDRsWM5daO/UzG+o8CHJpqMO1MRCUpLZz7NixtHPnzqgBvP7662Oz0SDwP3Ma6QxKHf+mTZuiO6hjjCTJgLBNazzovknKZWulxnByx4JGMEf77AwCT7Do81/+nY8jqGzvFCK+JzvT1JkQBH766afRBtxFV5LaFydeZIdw+kVGCBuEbDKSGkkKZzGD2EsVax6pqdTpZ+od2UilIQzXgXpAav79/ZAkA8J2w6wnOnLm0oG0M+PUsaqqKr3wwgtp165dqbGx0V92SSoxdAnlZIxB9QRK9913XzRwKSajpaMRAHLyd++990YNPJ2qSZ9lHbI+XZIMCEsChel1dXVp7NixRQ8YLoXdV04lqTlk+DxDjklT/e677+Ik0MVXkjoH0veZ5/fHH3/EPfztt9+OrqF0EGW9osaOWYVs+DHYvqysrF2awbDO8H3IeuH7Un/Oid+QIUMiBZYSCmolX3nllRiDxFgK5ieSmWNJgiQZEJZ8YEhHOE4MSWXpiHbahQ4wZtYTNYiMn6BF+JdffhkPEv5CS1LXDRgJFjlRZGzEunXr4gSONay6ujoCs9tuuy2NGzcuat0ZTcRJHesbwSQBHEPi2UAksOTP/BsnkqSuMneWzyO75I477oiO25zy0TWV70OaKw3QaLD222+/OZZIkgwIu47Tp09HLcPrr78eiyC7r+y8EiB2VNvwTH0iaUOk3wwaNChNmjQpFuWtW7emw4cPx4OBtYCSZKBI6in1iaRl0iyMDtu//vpramhoiPWNE0dGCh06dChGOBBU8mf+je7ZjDkiyOPzaIJDhglfj41GM00kyYCw22GkBAPglyxZEimYmfl/BGZtNXSXuVDl5eWRfjNx4sSYvcRwYXZjCfzciZUkSZJkQNjOmpqaYuf0wIED0ZyF4nhGSDzwwANp/Pjxafjw4VG7x0B70nAIGjMjJKhN5KSRLp98DB/L3L/KysoYg8EYCgbBE/TV19fHri27uXRdc0dWkiRJkgGhJEmSJMmAUJIkSZJkQChJkiRJMiCUJEmSJBkQSpIkSZIMCCVJkiRJBoSSJEmSJANCSZIkSZIBoSRJkiTJgFCSJEmSDAglSZIkSQaEkiRJkiQDQkmSJEmSAaEkSZIkyYBQkiRJkmRAKEmSJEkyIJQkSZIkGRBKkiRJkgwIJUmSJEkGhJIkSZIkA0JJkiRJkgGhJEmSJMmAUJIkSZJkQChJkiRJMiCUJEmSJBkQSpIkSZIMCCVJkiRJBoSSJEmSJANCSZIkSZIBoSRJkiTJgFCSJEmS9H/+AYh11S+FDRUzAAAAAElFTkSuQmCC"
```

Comment:
either user some small image value or break this line to multiple lines so that it doesn't exceed 119 characters limit.

Reply:
Done